packages/graphql-tag-pluck/package.json

```diff
@@ -29,13 +29,16 @@
     "graphql": "^14.0.0 || ^15.0.0 || ^16.0.0"
   },
   "dependencies": {
-    "@babel/parser": "7.16.8",
-    "@babel/traverse": "7.16.8",
-    "@babel/types": "7.16.8",
+    "@babel/parser": "^7.16.8",
+    "@babel/traverse": "^7.16.8",
+    "@babel/types": "^7.16.8",
     "@graphql-tools/utils": "^8.5.1",
     "tslib": "~2.3.0"
   },
   "devDependencies": {
+    "@babel/parser": "7.16.8",
+    "@babel/traverse": "7.16.8",
+    "@babel/types": "7.16.8",
```

Comment:
I think we can emove the dev dependencies here as they are redundant?

Reply:
I was thinking if this ran the test suite or transpiled itself, the versions could be pinned exactly.